tests/sentry/issues/test_ignored.py

```diff
@@ -48,10 +53,38 @@ def test_ignored_user_count(self) -> None:
 
 @apply_feature_flag_on_cls("organizations:escalating-issues")
 class HandleArchiveUntilEscalating(TestCase):  # type: ignore
-    def test_archive_until_escalating(self) -> None:
-        group = self.create_group()
-        add_group_to_inbox(group, GroupInboxReason.NEW)
+    @patch("sentry.issues.forecasts.query_groups_past_counts", return_value={})
+    def test_archive_until_escalating_no_counts(
+        self, mock_query_groups_past_counts: MagicMock
+    ) -> None:
+        self.group = self.create_group()
+        add_group_to_inbox(self.group, GroupInboxReason.NEW)
+
+        handle_archived_until_escalating([self.group], self.user)
+        assert not GroupInbox.objects.filter(group=self.group).exists()
+        assert not GroupSnooze.objects.filter(group=self.group).exists()
+
+        fetched_forecast = EscalatingGroupForecast.fetch(self.group.project.id, self.group.id)
+        assert fetched_forecast is None
+
+    @patch("sentry.issues.forecasts.query_groups_past_counts")
+    def test_archive_until_escalating_with_counts(
+        self, mock_query_groups_past_counts: MagicMock
+    ) -> None:
+        self.group = self.create_group()
+
+        mock_query_groups_past_counts.return_value = get_mock_groups_past_counts_response(
+            num_days=7, num_hours=1, groups=[self.group]
+        )
+
+        add_group_to_inbox(self.group, GroupInboxReason.NEW)
+
+        handle_archived_until_escalating([self.group], self.user)
+        assert not GroupInbox.objects.filter(group=self.group).exists()
+        assert not GroupSnooze.objects.filter(group=self.group).exists()
 
-        handle_archived_until_escalating([group.id], self.user)
```

Comment:
How was this working before with a list of ids instead of a list of groups?

<img width="232" alt="image" src="https://user-images.githubusercontent.com/44410/233410051-a42cc00f-0d05-4a22-8301-13debb5ac010.png">

Reply:
I think there's something weird going on with the group_list and group_ids variables - I'd like to look into what's going on later on in the refactor